stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/Toolbar.kt

```diff
@@ -40,7 +40,7 @@ fun Toolbar(
     from: String,
     to: String?,
     themeColor: Color,
-    navigationAction: () -> Unit,
+    launchBottomSheet: () -> Unit,
```

Comment:
does this always launch the bottom sheet or only if has data entered?. Maybe you can go from here to the viewmodel, check there the status and then decide if go back or launch the bottom sheet. Also, with this you don't need to expose the `canReview()` method from the viewModel. What do you think?